Explorer/Assets/DCL/Utilities/WorldProxy.cs

```diff
@@ -0,0 +1,42 @@
+using Arch.Core;
+
+namespace DCL.Utilities
+{
+    public class WorldProxy
```

Comment:
Considering we follow `#nullable` notation there are a lot of warnings. Can you fix them?

Reply:
Done [in this commit](https://github.com/decentraland/unity-explorer/pull/249/commits/e8b4869f07a8147fba8e1fc07d5f841f4809d4fc)